iris/index.js

```diff
@@ -21,6 +21,51 @@ import * as data from './data';
 import * as loader from './loader';
 import * as ui from './ui';
 
+// TODO(cais): Remove in favor of tf.confusionMatrix once it's available.
+//   https://github.com/tensorflow/tfjs/issues/771
+/**
+ * Calcualte the confusion matrix.
+ *
+ * @param {tf.Tensor} labels The target labels, assumed to be 0-based integers
+ *   for the categories. The shape is `[numExamples]`, where
+ *   `numClasses` is the number of possible classes.
+ * @param {tf.Tensor} predictions The predicted probabilities, assumed to be
+ *   0-based integers for the categories. Must have the same shape as `labels`.
+ * @param {number} numClasses Number of all classes, if not provided,
+ *   will calculate from both `labels` and `predictions`.
+ * @return {tf.Tensor} The confusion matrix as a 2D tf.Tensor. The rows
+ *   correspond to the truth classes and the columns correspond to the
+ *   predicted classes.
```

Comment:
The values correspond to the number of times true class ROW was predicted as class COL.